apps/emqx_cluster_link/src/emqx_cluster_link_router_syncer.erl

```diff
@@ -98,20 +105,45 @@ do_push(SyncerName, OpName, Topic, ID) ->
             dropped
     end.
 
+unlink_remote(TargetCluster, Reason) ->
```

Comment:
(Related to the my previous rant) Nit. Sort of weird to have such responsibility in a module that deals with route replication.

Reply:
Removed